examples/quickstart/non_atomic_rma.cpp

```diff
@@ -77,10 +76,8 @@ int hpx_main(po::variables_map &vm)
     int result = 0;
     double elapsed = 0.0;
 
-    components::component_type type =
-       components::get_component_type<components::server::plain_function<set_initialdata_action> >();
     std::vector<naming::id_type> localities =
-        hpx::find_remote_localities(type);
+        hpx::find_remote_localities();
 
```

Comment:
Is there still a way to identify plain actions for selecting localities supporting them?